### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by fredO13** [open-mmlab/mmrotate#776]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmrotate/issues) and [Discussions](https://github.com/open-mmlab/mmrotate/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmrotate.readthedocs.io/en/1.x/notes/4_faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmrotate) or [latest version (1.x)](https://github.com/open-mmlab/mmrotate/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmrotate

### Environment

# packages in environment at E:\FredO\anaconda\envs\mmrotate:
#
# Name                    Version                   Build  Channel
absl-py                   0.15.0             pyhd3eb1b0_0    anaconda
addict                    2.4.0                    pypi_0    pypi
aiohttp                   3.8.1            py38h2bbff1b_1    anaconda
aiosignal                 1.2.0              pyhd3eb1b0_0    anaconda
argon2-cffi               21.3.0             pyhd3eb1b0_0    anaconda
argon2-cffi-bindings      21.2.0           py38h2bbff1b_0    anaconda
asttokens                 2.0.5              pyhd3eb1b0_0    anaconda
async-timeout             4.0.1              pyhd3eb1b0_0    anaconda
attrs                     21.4.0             pyhd3eb1b0_0    anaconda
backcall                  0.2.0              pyhd3eb1b0_0    anaconda
beautifulsoup4            4.11.1           py38haa95532_0    anaconda
blas                      1.0                         mkl    anaconda
bleach                    4.1.0              pyhd3eb1b0_0    anaconda
blinker                   1.4              py38haa95532_0    anaconda
brotlipy                  0.7.0           py38h2bbff1b_1003    anaconda
ca-certificates           2022.4.26            haa95532_0    anaconda
cachetools                4.2.2              pyhd3eb1b0_0    anaconda
certifi                   2022.6.15        py38haa95532_0    anaconda
cffi                      1.15.0           py38h2bbff1b_1    anaconda
charset-normalizer        2.0.4              pyhd3eb1b0_0    anaconda
click                     8.1.3                    pypi_0    pypi
colorama                  0.4.6                    pypi_0    pypi
contourpy                 1.0.7                    pypi_0    pypi
cryptography              37.0.1           py38h21b164f_0    anaconda
cudatoolkit               11.3.1               h59b6b97_2    anaconda
cycler                    0.11.0                   pypi_0    pypi
dataclasses               0.8                pyh6d0b6a4_7    anaconda
debugpy                   1.5.1            py38hd77b12b_0    anaconda
decorator                 5.1.1              pyhd3eb1b0_0    anaconda
defusedxml                0.7.1              pyhd3eb1b0_0    anaconda
e2cnn                     0.2.3                    pypi_0    pypi
entrypoints               0.4              py38haa95532_0    anaconda
executing                 0.8.3              pyhd3eb1b0_0    anaconda
fonttools                 4.39.2                   pypi_0    pypi
freetype                  2.10.4                   vc14_3  [vc14]  esri
frozenlist                1.2.0            py38h2bbff1b_0    anaconda
google-auth               2.6.0              pyhd3eb1b0_0    anaconda
google-auth-oauthlib      0.4.1                      py_2    anaconda
grpcio                    1.42.0           py38hc60d5dd_0    anaconda
icu                       58.2             vc14hc45fdbb_0  [vc14]  anaconda
idna                      3.3                pyhd3eb1b0_0    anaconda
importlib-metadata        6.1.0                    pypi_0    pypi
importlib-resources       5.12.0                   pypi_0    pypi
importlib_resources       5.2.0              pyhd3eb1b0_1    anaconda
intel-openmp              2021.4.0          haa95532_3556    anaconda
ipykernel                 6.9.1            py38haa95532_0    anaconda
ipython                   8.3.0            py38haa95532_0    anaconda
ipython_genutils          0.2.0              pyhd3eb1b0_1    anaconda
ipywidgets                7.6.5              pyhd3eb1b0_1    anaconda
jedi                      0.18.1           py38haa95532_1    anaconda
jinja2                    3.0.3              pyhd3eb1b0_0    anaconda
jpeg                      9b               vc14h4d7706e_1  [vc14]  anaconda
jsonschema                4.4.0            py38haa95532_0    anaconda
jupyter                   1.0.0                    py38_7    anaconda
jupyter_client            7.2.2            py38haa95532_0    anaconda
jupyter_console           6.4.3              pyhd3eb1b0_0    anaconda
jupyter_core              4.10.0           py38haa95532_0    anaconda
jupyterlab_pygments       0.1.2                      py_0    anaconda
jupyterlab_widgets        1.0.0              pyhd3eb1b0_1    anaconda
kiwisolver                1.4.4                    pypi_0    pypi
libpng                    1.6.37               h2a8f88b_0    anaconda
libprotobuf               3.19.1               h23ce68f_0    anaconda
libtiff                   4.2.0                hd0e1b90_0    anaconda
libuv                     1.40.0               he774522_0    anaconda
libwebp                   1.2.2                h2bbff1b_0    anaconda
lz4-c                     1.9.3                h2bbff1b_1    anaconda
markdown                  3.4.1                    pypi_0    pypi
markdown-it-py            2.2.0                    pypi_0    pypi
markupsafe                2.1.1            py38h2bbff1b_0    anaconda
matplotlib                3.7.1                    pypi_0    pypi
matplotlib-inline         0.1.2              pyhd3eb1b0_2    anaconda
mdurl                     0.1.2                    pypi_0    pypi
mistune                   0.8.4           py38he774522_1000    anaconda
mkl                       2021.4.0           haa95532_640    anaconda
mkl-service               2.4.0            py38h2bbff1b_0    anaconda
mkl_fft                   1.3.1            py38h277e83a_0    anaconda
mkl_random                1.2.2            py38hf11a4ad_0    anaconda
mmcv-full                 1.7.1                    pypi_0    pypi
mmdet                     2.28.2                   pypi_0    pypi
mmrotate                  0.3.4                    pypi_0    pypi
model-index               0.1.11                   pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
multidict                 5.1.0            py38h2bbff1b_2    anaconda
nbclient                  0.5.13           py38haa95532_0    anaconda
nbconvert                 6.4.4            py38haa95532_0    anaconda
nbformat                  5.3.0            py38haa95532_0    anaconda
nest-asyncio              1.5.5            py38haa95532_0    anaconda
notebook                  6.4.11           py38haa95532_0    anaconda
numpy                     1.22.3           py38h7a0a035_0    anaconda
numpy-base                1.22.3           py38hca35cd5_0    anaconda
oauthlib                  3.1.0                      py_0    anaconda
opencv-python             4.7.0.72                 pypi_0    pypi
openmim                   0.3.6                    pypi_0    pypi
openssl                   1.1.1o               h2bbff1b_0    anaconda
ordered-set               4.1.0                    pypi_0    pypi
packaging                 23.0                     pypi_0    pypi
pandas                    1.5.3                    pypi_0    pypi
pandocfilters             1.5.0              pyhd3eb1b0_0    anaconda
parso                     0.8.3              pyhd3eb1b0_0    anaconda
pickleshare               0.7.5           pyhd3eb1b0_1003    anaconda
pillow                    9.0.1            py38hdc2b20a_0    anaconda
pip                       21.2.2           py38haa95532_0    anaconda
prometheus_client         0.13.1             pyhd3eb1b0_0    anaconda
prompt-toolkit            3.0.20             pyhd3eb1b0_0    anaconda
prompt_toolkit            3.0.20               hd3eb1b0_0    anaconda
protobuf                  3.19.1           py38hd77b12b_0    anaconda
pure_eval                 0.2.2              pyhd3eb1b0_0    anaconda
pyasn1                    0.4.8              pyhd3eb1b0_0    anaconda
pyasn1-modules            0.2.8                      py_0    anaconda
pycocotools               2.0.6                    pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0    anaconda
pygments                  2.14.0                   pypi_0    pypi
pyjwt                     1.7.1                    py38_0    anaconda
pyopenssl                 22.0.0             pyhd3eb1b0_0    anaconda
pyparsing                 3.0.9                    pypi_0    pypi
pyqt                      5.9.2            py38ha925a31_4    anaconda
pyrsistent                0.18.0           py38h196d8e1_0    anaconda
pysocks                   1.7.1            py38haa95532_0    anaconda
python                    3.8.13               h6244533_0    anaconda
python-dateutil           2.8.2              pyhd3eb1b0_0    anaconda
python-fastjsonschema     2.15.1             pyhd3eb1b0_0    anaconda
pytorch                   1.12.1          py3.8_cuda11.3_cudnn8_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2022.7.1                 pypi_0    pypi
pywin32                   302              py38h2bbff1b_2    anaconda
pywinpty                  2.0.2            py38h5da7b33_0    anaconda
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     22.3.0           py38hd77b12b_2    anaconda
qt                        5.9.7            vc14h73c81de_0  [vc14]  anaconda
qtconsole                 5.3.0              pyhd3eb1b0_0    anaconda
qtpy                      2.0.1              pyhd3eb1b0_0    anaconda
regex                     2022.10.31               pypi_0    pypi
requests                  2.27.1             pyhd3eb1b0_0    anaconda
requests-oauthlib         1.3.0                      py_0    anaconda
rich                      13.3.2                   pypi_0    pypi
rsa                       4.7.2              pyhd3eb1b0_1    anaconda
scipy                     1.10.1                   pypi_0    pypi
send2trash                1.8.0              pyhd3eb1b0_1    anaconda
setuptools                61.2.0           py38haa95532_0    anaconda
sip                       6.5.1            py38hd77b12b_0    anaconda
six                       1.16.0             pyhd3eb1b0_1    anaconda
soupsieve                 2.3.1              pyhd3eb1b0_0    anaconda
sqlite                    3.38.5               h2bbff1b_0    anaconda
stack_data                0.2.0              pyhd3eb1b0_0    anaconda
sympy                     1.11.1                   pypi_0    pypi
tabulate                  0.9.0                    pypi_0    pypi
tensorboard               2.6.0                      py_1    anaconda
tensorboard-data-server   0.6.0            py38haa95532_0    anaconda
tensorboard-plugin-wit    1.6.0                      py_0    anaconda
terminado                 0.13.1           py38haa95532_0    anaconda
terminaltables            3.1.10                   pypi_0    pypi
testpath                  0.6.0            py38haa95532_0    anaconda
tk                        8.6.7            vc14hb68737d_1  [vc14]  anaconda
toml                      0.10.2             pyhd3eb1b0_0    anaconda
torchaudio                0.12.1               py38_cu113    pytorch
torchvision               0.13.1               py38_cu113    pytorch
tornado                   6.1              py38h2bbff1b_0    anaconda
traitlets                 5.1.1              pyhd3eb1b0_0    anaconda
typing-extensions         4.1.1                hd3eb1b0_0    anaconda
typing_extensions         4.1.1              pyh06a4308_0    anaconda
urllib3                   1.26.9           py38haa95532_0    anaconda
vc                        14.2                 h21ff451_1    anaconda
vs2015_runtime            14.27.29016          h5e58377_2    anaconda
wcwidth                   0.2.5              pyhd3eb1b0_0    anaconda
webencodings              0.5.1                    py38_1    anaconda
werkzeug                  2.0.3              pyhd3eb1b0_0    anaconda
wheel                     0.37.1             pyhd3eb1b0_0    anaconda
widgetsnbextension        3.5.2            py38haa95532_0    anaconda
win_inet_pton             1.1.0            py38haa95532_0    anaconda
wincertstore              0.2              py38haa95532_2    anaconda
winpty                    0.4.3                         4    anaconda
xz                        5.2.5                h8cc25b3_1    anaconda
yapf                      0.32.0                   pypi_0    pypi
yarl                      1.6.3            py38h2bbff1b_0    anaconda
zipp                      3.15.0                   pypi_0    pypi
zlib                      1.2.11           vc14h1cdd9ab_1  [vc14]  anaconda
zstd                      1.4.5                h04227a9_0    anaconda

### Reproduces the problem - code sample

train_detector(model, datasets, cfg, distributed=False, validate=True)

### Reproduces the problem - command or script

see above. 
training seems to start than it seems to start a second time and everything crashes.
I tried this cfg.mp_start_method = None but it doesn't change anything

### Reproduces the problem - error message

\spawn.py", line 134, in _check_not_importing_main
    raise RuntimeError('''
RuntimeError:
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

**Comment by zytx121** [comment]
Hi @fredO13, the maintenance of the old version has stopped. Welcome to the new version: https://github.com/open-mmlab/mmrotate/tree/1.x
